Lib9c/Model/State/WeeklyArenaState.cs

```diff
@@ -100,6 +100,11 @@ private void ResetOrderedArenaInfos()
             int firstRank = 1,
             int? count = null)
         {
+            if (OrderedArenaInfos.Count == 0)
+            {
+                return new List<(int rank, ArenaInfo arenaInfo)>();
+            }
+
```

Comment:
I guess it would be better if we have a regression test case for it.

Reply:
Added it!